BlogTemplate/wwwroot/css/site.css

```diff
@@ -1,7 +1,11 @@
-﻿body {
+﻿@import "colors.css";
+
+body {
     margin: 0;
     padding: 0;
-    font: 16px/1.5 arial;
+    font-family: Garamond, 'Baskerville Old Face', Georgia, 'Times New Roman', serif;
```

Comment:
Since we're going down the route for CSS custom declarations (i.e. variables), we could do the same for font as well so that you don't have to repeat this string.